settings.py

```diff
@@ -119,7 +119,7 @@
 HTTP_GA_SRC = 'http://www.google-analytics.com'
 CSP_FRAME_SRC += ('https://www.sandbox.paypal.com',)
 CSP_IMG_SRC += (HTTP_GA_SRC,)
-CSP_SCRIPT_SRC += (HTTP_GA_SRC,)
+CSP_SCRIPT_SRC += (HTTP_GA_SRC, "'self'")
```

Comment:
I am just forgetful of the settings maze here. This is for local dev only and _won't_ be used elsewhere (prod where CDN is used), right?